client/js/screens/Home/Home.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {Text} from 'react-native';
+
+const Home = () => {
```

Comment:
add props as arg for your presentation component.

Reply:
completed.